requirements.in

```diff
@@ -23,9 +23,6 @@ kv
 loguru
 mkdocs-material>=5.3.0
 mkdocs
-ogc-plugins-juju>=1.0.39
-ogc-plugins-runner>=1.0.24
-ogc>=1.0.1
```

Comment:
`ogc-plugins-juju` and `ogc-plugins-runner` were [removed from pypi](https://github.com/adam-stokes/ogc/issues/11) by Mr. Stokes and to mitigate this loss, will purge these from the requirements.